src/server/command/index.ts

```diff
@@ -24,14 +24,16 @@ export default (
       conn: { remoteAddress },
     },
   }: Socket,
-  { user, host, port, auth, pass, key }: SSH,
-  command: string
+  { user, askuser, host, port, auth, pass, key }: SSH,
+  command: string,
+  forcessh: boolean
 ): { args: string[]; user: boolean } => ({
-  args: localhost(host)
+  args: !forcessh && localhost(host)
     ? loginOptions(command, remoteAddress)
     : sshOptions(
         urlArgs(referer, {
-          host: address(referer, user, host),
+          sshcommand: askuser ? './bin/ssh-with-user' : 'ssh',
```

Comment:
Can we avoid using a bin wrapper, we previously did this and it created a large number of path issues

Reply:
Sure. I made it without the whole change of the user asking...